ci-operator/config/openshift/terraform-providers/openshift-terraform-providers-main.yaml

```diff
@@ -0,0 +1,54 @@
+base_images:
+  base:
+    name: "4.15"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.20-openshift-4.15:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.15
+binary_build_commands: make
+build_root:
+  from_repository: true
```

Comment:
Not a blocking point, but personally I think it would be more convenient without `from_repository: true`, but preferring a direct configuration directly here (as mentioned in the [docs](https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-root-image)), rather than into an external file located in the repo.

Reply:
Doesn't that make things trickier when we want to update the Dockerfile itself?

Reply:
Should be better now.

Reply:
I think that having `.ci.yaml` in a repo may be more convenient in those situations where you'd want to bump the build root image _and_ change the content of the image repo Dockerfile at the same time (so with just one PR). In pratice, (and that's more on a personal point of view) I very rarely had that need, while I found confusing every time I wanted to check the base root to jump from the CI configuration to the repo one.